### ISSUE TITLE
yml files have different icons

### ISSUE THREAD

**Issue by Stanzilla** [file-icons/vscode#54]
![image](https://user-images.githubusercontent.com/75278/85473901-34098000-b5b4-11ea-8b20-75fb4431b821.png)

any reason why? does the file name matter?

**Comment by Alhadis** [comment]
> does the file name matter?

Yes, it does. The icon you're seeing for `mkdocs.yml` is being used to represent MkDocs-related files. The reason you're not seeing it for the `mkdocs-prod.yml` file is because that filename hasn't been registered. Are there any other filenames that should be receiving the MkDocs icon?

**Comment by Stanzilla** [comment]
Well in theory everything with mkdocs in the name then, there is no special rule on what you can or can't have in the file name which makes this a bit more complicated I guess

**Comment by ajotaos** [comment]
Does VSCode allow something patterns like `mkdocs-*.yml`?

**Comment by Alhadis** [comment]
> Does VSCode allow something patterns like `mkdocs-*.yml`?

No, it doesn't. And it's a _major_ pain-in-the-arse.
